### PR TITLE
ios logs

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

File: src/cli/cli.cc [hunk 1]
```diff
@@ -383,6 +383,18 @@ void runIOSSimulator (const fs::path& path, Map& settings) {
     exit(rInstallApp.exitCode);
   }
 
+  log("launching the child logger process");
```
Comment (jwerle) [inline src/cli/cli.cc:386]: maybe we output this for debug only?

Comment (chicoxyzzy) [reply src/cli/cli.cc:386]: Yes, I don't think we need it. The purpose of this PR is to save a draft of how to enable logging. Currently, streaming doesn't work for me, I only see a first message from `log stream` about starting to listen to the log

**Comment by chicoxyzzy** [comment]
there's also a `xcrun simctl spawn booted log stream` command which I believe will only watch for the booted device

File: src/cli/cli.cc (hunk 1 above)
Comment (jwerle) [reply src/cli/cli.cc:386]: yep log stream only works for local stores

**Pull request closed without merging**